docs/source/guides/configuration.rst

```diff
@@ -0,0 +1,51 @@
+Configuration
+=============
+
+In this section we will understand how to configure Iroha. Let's take a look
+at ``example/config.sample``
+
+.. code-block:: json
+  :linenos:
+
+  {
+    "block_store_path": "/tmp/block_store/",
+    "torii_port": 50051,
+    "internal_port": 10001,
+    "pg_opt": "host=localhost port=5432 user=postgres password=mysecretpassword",
+    "max_proposal_size": 10,
+    "proposal_delay": 5000,
+    "vote_delay": 5000,
+    "load_delay": 5000
+  }
+
+As you can see, configuration file is a valid ``json`` structure. Let's go 
+line-by-line and understand what every parameter means.
+
+Deployment-specific parameters
+------------------------------
+
+- ``block_store_path`` sets path to the folder where blocks will be stored
+- ``torii_port`` sets the port for external communications. Queries and
+  transactions are sent here.
+- ``internal_port`` sets the port for internal communications: ordering
+  service, consensus and block loader.
+- ``pg_opt`` is used for setting credentials of PostgreSQL: hostname, port,
+  username and password.
+
+Environment-specific parameters
+-------------------------------
+
+- ``max_proposal_size`` is the maximum  amount of transactions that can be in
+  one proposal. For a starter you can stick to ``10``. However, we recommend
+  to increase this number if you have a lot of transactions per second.
+- ``proposal_delay`` is a maximum waiting time in milliseconds before emitting
+  a new proposal. Proposal is emitted if the ``max_proposal_size`` is reached 
+  or ``proposal_delay`` milliseconds had passed. You can start with ``5000``
+  and increase this number if you have a lot of transactions per second.
```

Comment:
since it is likely that with an intense load (over 100 transactions per second) and low value of proposal delay there will be many proposals of small size.